### ISSUE TITLE
queryAssignedElements decorator not included in import map

### ISSUE THREAD

**Issue by zeroedin** [patternfly/patternfly-elements#2137]
When using [@queryAssignedElements](https://lit.dev/docs/api/decorators/#queryAssignedElements) you currently get the following error on the [demo page](https://deploy-preview-2099--patternfly-elements.netlify.app/components/tabs/demo/): 

```
Uncaught SyntaxError: The requested module 'lit/decorators.js' does not provide an export named 'queryAssignedElements'
```

I believe the issue is caused by `docs/_data/importMap.cjs` not including an import for that lit module.

Version 2.0.2 of lit does not have the decorator as an export, version 2.1.0 is the oldest version that does, not sure what all the ramifications are for changing any of the version numbers in this file.

**Comment by bennypowers** [comment]
That file only affects the eleventy build, so feel free to update it however you want. Might be nice to use a semver specifier like `lit@^2`

**Comment by zeroedin** [comment]
Closed in #2143
